### PR TITLE
FORGE-2093: Being able to create a new JPA listener

### PR DESCRIPTION
**Pull request by agoncal**

### CODE REVIEW

File: javaee/impl/src/main/java/org/jboss/forge/addon/javaee/jpa/ui/JPANewEntityListenerCommand.java [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+   public void initializeUI(UIBuilder builder) throws Exception
+   {
+      super.initializeUI(builder);
+      builder.add(lifecycles);
+   }
+
+   public JavaClassSource decorateSource(UIExecutionContext context, Project project, JavaClassSource source)
+            throws Exception
+   {
+      Collection<LifecycleType> lifecycles = Iterators.asList(this.lifecycles.getValue());
```
Comment (gastaldi) [inline javaee/impl/src/main/java/org/jboss/forge/addon/javaee/jpa/ui/JPANewEntityListenerCommand.java]: No need to convert to list if you just want to iterate

Comment (agoncal) [inline javaee/impl/src/main/java/org/jboss/forge/addon/javaee/jpa/ui/JPANewEntityListenerCommand.java]: Fixed

Comment (gastaldi) [inline javaee/impl/src/main/java/org/jboss/forge/addon/javaee/jpa/ui/JPANewEntityListenerCommand.java]: Hi @agoncal, could you create a new PR with this fix? Thanks

Comment (gastaldi) [inline javaee/impl/src/main/java/org/jboss/forge/addon/javaee/jpa/ui/JPANewEntityListenerCommand.java]: On a second thought, I just changed it and committed, no worries.

**Pull request merged**